spec/xliffer/string_spec.rb

```diff
@@ -98,6 +118,27 @@ module XLIFFer
       end
     end
 
+    describe '#source_components' do
+      it 'Distinguishes un-tagged from tag-labeled sections' do
+        xml = <<-XML
+        <trans-unit id="my id with html">
+          <source>&lt;Hello&gt;<ph>&lt;br&gt;</ph>World</source>
+          <target>&lt;Bonjour&gt;<ph>&lt;br&gt;</ph>le monde</target>
+        </trans-unit>
+        XML
+        trans_unit_node = Nokogiri::XML.parse(xml).xpath('//trans-unit').first
+        puts trans_unit_node
```

Comment:
And this line too 🙇

Reply:
Also removed in [68cefdf](https://github.com/WOVNio/xliffer/pull/8/commits/68cefdfb3fd411803d36d3aa30c1bdfe38011332)  🙇

Reply:
Thank you!